src/lib/libhtml5.js

```diff
@@ -1595,10 +1595,9 @@ var LibraryHTML5 = {
     '$specialHTMLTargets'
 #endif
   ],
-  emscripten_set_pointerlockchange_callback_on_thread__docs: '/** @suppress {missingProperties} */', // Closure does not see document.body.mozRequestPointerLock etc.
   emscripten_set_pointerlockchange_callback_on_thread: (target, userData, useCapture, callbackfunc, targetThread) => {
     // TODO: Currently not supported in pthreads or in --proxy-to-worker mode. (In pthreads mode, document object is not defined)
-    if (!document || !document.body || (!document.body.requestPointerLock && !document.body.mozRequestPointerLock && !document.body.webkitRequestPointerLock && !document.body.msRequestPointerLock)) {
+    if (!document || !document.body?.requestPointerLock) {
```

Comment:
`!document || !document.body?.requestPointerLock` is logically equivalent to `!(document && document.body?.requestPointerLock)`, and in that form it's easy to see it's equivalent to `!document?.body?.requestPointerLock`

Reply:
True, and also all these functions are marked as `__proxy: sync` so document itself should always exist.

Reply:
Fixed